lib/plugins/blocks.js

```diff
@@ -19,13 +19,13 @@ function inject (bot, { version }) {
   const Block = require('prismarine-block')(version)
   const Chunk = require('prismarine-chunk')(version)
   const ChatMessage = require('../chat_message')(version)
-  let columns = {}
   const signs = {}
   const paintingsByPos = {}
   const paintingsById = {}
-
   const blockEntities = new Map()
 
+  let columns = {}
```

Comment:
```suggestion
  const columns = {}
```
i haven't seen this be re-assigned anywhere, so it can be a const

Reply:
L479